code/modules/hydroponics/grown/cannabis.dm

```diff
@@ -79,6 +79,34 @@
 						"lipolicide" = 0.15)
 	rarity = 69
 
+/obj/item/seeds/cannabis/ultimate/harvest
+	harvest(mob/user)
```

Comment:
umm?

Reply:
Oh I guess that was uninteded as well? I remember changing something about harvest but it didn't work well, let me test and I will give an answer in a sec

Reply:
this is not the typical syntax we use for procs.

Reply:
Well, it works, but it does look bad, and removing it make the cannabis just go up to 50 unit, I tried adding max_volume = 420 to it, but it then used the new way to calculate reagents?!?!

Reply:
If you have to leave the original implementation in for this specific plant because of a joke the joke is probably not worth having in it's current iteration.

also fix the syntax still lol